app/src/app/decret-form/decret-form.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { ALL_DECREE } from '../../../../common/config';
+import { AngularFireDatabase, snapshotChanges } from '@angular/fire/database';
+import { AngularFireAuth } from '@angular/fire/auth';
+import { MatDialog } from '@angular/material';
+import { take, tap } from 'rxjs/operators';
+import { AvailableQuestion } from '../model';
+
+@Component({
+  selector: 'app-decret-form',
+  templateUrl: './decret-form.component.html',
+  styleUrls: ['./decret-form.component.css']
+})
+export class DecretFormComponent implements OnInit {
+
+  constructor(private fb: FormBuilder, private db: AngularFireDatabase, private auth: AngularFireAuth, private dialog: MatDialog) { }
+
+  decretForm: FormGroup;
+  title: string
+  state: string;
+
+  allDecree = ALL_DECREE
+  availableDecree: String[]
+
+  @Input()
+  house: AvailableQuestion
```

Comment:
Strange naming - `house` and type `AvailableQuestion`

Reply:
Yeah 😄 , I will make it less strange.

Reply:
Fixed.